webrender/res/ps_text_run.glsl

```diff
@@ -160,16 +163,13 @@ void main(void) {
 void main(void) {
     vec3 tc = vec3(clamp(vUv.xy, vUvBorder.xy, vUvBorder.zw), vUv.z);
     vec4 mask = texture(sColor0, tc);
+    mask.rgb = mask.rgb * vMaskSwizzle.x + mask.a * vMaskSwizzle.y;
```

Comment:
I think `mix(mask.rgb, mask.aaa, vMaskSwizzle)` would look cleaner, and also reflect the fact you only need one value in the `vMaskSwizzle` instead of two.

Reply:
This is not true for `MODE_SUBPX_BG_PASS1` which uses `vec2(-1.0, 1.0)` as the swizzle.

Reply:
Good point! It's technically not a swizzle then.